index.html

```diff
@@ -11,7 +11,7 @@ <h1>Task Lister Lite™️</h1>
 
     <form id="create-task-form" action="" method="POST">
       <label for="new-task-description">Task description:</label>
-      <input type="text" id="new-task-description" name="new-task-description" placeholder="description">
+      <input type="text" id="newtaskdescription" name="new-task-description" placeholder="description">
```

Comment:
I don't think we want to change this - it's best practice to separate words by dashes in HTML, to my knowledge.

You'd just need to update your solution code to use bracket notation instead of dot notation. (See comment on that file.)